objects/client.c

```diff
@@ -371,14 +371,33 @@
  */
 
 /**
- * The client icon.
+ * The client icon as a surface (mapped memory).
```

Comment:
What is `mapped memory` supposed to mean? It confuses me here. It's a normal image surface, but I'd consider this an implementation detail.